### ISSUE TITLE
SSH Host autocompletion with Include directive

### ISSUE THREAD

**Issue by dobegor** [fish-shell/fish-shell#3833]
OpenSSH now supports `include` directive.
```
Include
         Include the specified configuration file(s).  Mul‐
         tiple pathnames may be specified and each pathname
         may contain glob(3) wildcards and, for user config‐
         urations, shell-like “~” references to user home
         directories.  Files without absolute paths are
         assumed to be in ~/.ssh if included in a user con‐
         figuration file or /etc/ssh if included from the
         system configuration file.  Include directive may
         appear inside a Match or Host block to perform con‐
         ditional inclusion.
```

Currently fish doesn't autocomplete SSH hosts from included configs.

**Comment by faho** [comment]
Already fixed by #3781, which will be included in the next release.

**Comment by dobegor** [comment]
thanks!
